cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -1432,16 +1569,41 @@ private void generateHttpOperations(
         reference = new Reference("#/channels/%s-item".formatted(name));
         operation.setChannel(reference);
         reference = new Reference("#/channels/%s-item/messages/%s".formatted(name, messageName));
-        operation.setMessages(Collections.singletonList(reference));
+        operation.setMessages(List.of(reference));
+        operation.setBindings(operationBindings);
+        if (secure)
+        {
+            operation.setSecurity(List.of(security));
+        }
+        operations.put("on%sReadItem".formatted(label), operation);
+
+        operation = new Operation();
+        operation.setAction(OperationAction.RECEIVE);
+        reference = new Reference("#/channels/%s".formatted(name));
+        operation.setChannel(reference);
+        reference = new Reference("#/channels/%s/messages/%ss".formatted(name, messageName));
+        operation.setMessages(List.of(reference));
         httpBinding = new HTTPOperationBinding();
         httpBinding.setMethod(GET);
-        sseBinding = new ZillaSseOperationBinding(GET);
-        operation.setBindings(Map.of("http", httpBinding, "x-zilla-sse", sseBinding));
+        operation.setBindings(Map.of("http", httpBinding));
         if (secure)
         {
             operation.setSecurity(List.of(security));
         }
-        operations.put("on%sReadItem".formatted(label), operation);
+        operations.put("on%sGet".formatted(label), operation);
+
+        operation = new Operation();
+        operation.setAction(OperationAction.RECEIVE);
+        reference = new Reference("#/channels/%s-item".formatted(name));
+        operation.setChannel(reference);
+        reference = new Reference("#/channels/%s-item/messages/%s".formatted(name, messageName));
+        operation.setMessages(List.of(reference));
+        operation.setBindings(Map.of("http", httpBinding));
+        if (secure)
+        {
+            operation.setSecurity(List.of(security));
+        }
+        operations.put("on%sGetItem".formatted(label), operation);
```

Comment:
This is the new operation to handle REST GET instead of SSE GET right?
Good workaround. We should also discuss if there is a cleaner way to share the same path for both, but still be able to detect properly which one to serve.